index.html

```diff
@@ -245,5 +245,41 @@ <h4>Skills</h4>
             </li>
         </ul>
     </section>
+    <!--Starting form section-->
+    <section>
+        <div class="contact-form">
+            <h3 class="form">I'm always interested in hearing about new projects, so if you would like to chat please</h3>
+            <div class="con-form">
+                <form action="#" method="post" class="request" >
+                    <input type="text" placeholder="Full name" maxlength="30" required>
+                    <input type="email" placeholder="Email address" maxlength="30" required>
+                    <textarea name="request" id="request" cols="30" rows="10" placeholder="Enter text area" maxlength="500" required></textarea>
+                    <div class="see mbili">
+                        <button type="submit">Get in touch</button>
```

Comment:
- [ ] You have done a tremendous job styling your submit button. However, so that you stick as much as possible to the chosen Figma design, kindly make sure that you implement the button states eg on hover, on submit etc.This is the missing piece to perfection.

**Below are the desired button states**
![image](https://user-images.githubusercontent.com/86318284/177387670-7dd21033-b8ff-41ab-a060-49c2d0a73154.png)